ha/ut/note.c

```diff
@@ -109,7 +113,7 @@ static struct {
 
 static void start_rpc_client_and_server(void)
 {
-	int rc;
+	int                 rc;
```

Comment:
No need to have so many spaces between "int" and "rc".

Reply:
It is observed that the below pkill command is not able to kill the server process.
Hence added this to retry to kill the process during cleanup